src/wechaty_puppet/memory_card/memorycardmultiplex_test.py

```diff
@@ -0,0 +1,79 @@
+import unittest
```

Comment:
I'd like to suggest we stay with using the pytest with all our Python programs.

Reply:
OK，got it! Thanks, I will recode the tests.